WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -40,6 +63,48 @@ - (BOOL)fb_goToHomescreenWithError:(NSError **)error
   return YES;
 }
 
+- (BOOL)fb_lockScreen:(NSError **)error
+{
+  if (locked) {
+    return TRUE;
+  }
+  SEL mSelector = NSSelectorFromString(@"pressLockButton");
```

Comment:
`performSelector` might be shorter. It is anyway not needed to pass any arguments there